test/keyshot_submitter/test_keyshot_submitter.py

```diff
@@ -242,3 +243,45 @@ def test_settings_apply_submitter_settings():
     assert sorted(settings.input_directories) == ["test_directory_2"]
     assert sorted(settings.output_directories) == ["test_directory_2", "test_directory_4"]
     assert sorted(settings.referenced_paths) == ["test_ref_path_2"]
+
+
+def test_save_ksp_bundle():
+    dir = os.path.normpath("/testdir/test")
+    bundle_name = "test_bundle.ksp"
+    expected_bundle_path = os.path.normpath(f"{dir}/{bundle_name}")
+
+    output = submitter._save_ksp_bundle(dir, bundle_name)
```

Comment:
nit: if we're testing this function, should it be a public function (without leading underscore)?

Reply:
Leftover from before my initial rebase. Removed the leading `_` from all new functions to follow the pattern we've already established in the file.